plugin/forward/README.md

```diff
@@ -39,7 +39,8 @@ forward FROM TO...
   limited to 15.
 
 Multiple upstreams are randomized (see `policy`) on first use. When a healthy proxy returns an error
-during the exchange the next upstream in the list is tried.
+during the exchange the next upstream in the list is tried. Metadata parameter "forward/resolving_proxy" 
+contains the ip-address and the port of the replied upstream on success, see the log plugin for detail.
```

Comment:
I think we should make a specific section **Metadata** the same way we have a **Metrics** section.
And list the each metadata information we are gathering in this plugin.

Something like:

### Metadata

* `forward/resolving_proxy` : provide the IP and port of the upstream resolver used to resolve current DNS query.

Reply:
Ok, added this section.